.scalafmt.conf

```diff
@@ -0,0 +1,8 @@
+style = IntelliJ
+maxColumn = 100
```

Comment:
I'm using 100 columns for the Scala code to match the Java code, which uses the Google formatting rules.